app/client/src/components/shared/MapMouseEvents.tsx

```diff
@@ -56,7 +55,7 @@ function getGraphicsFromResponse(
 ) {
   if (!res.results || res.results.length === 0) return null;
 
-  const matches = res.results.filter((result) => {
+  const matches = res.results.filter((result: any) => {
```

Comment:
Is there a particular reason to add the `any` type here? TypeScript infers the type fine without it on my end, and the type doesn't offer a benefit otherwise.

Reply:
In both of these cases its because the `__esri.HitTestResult` type doesn't include `graphic`. I suppose we could extend the type to include `graphic`.

Reply:
I ended up fixing this a different way. TypeScript infers `result` as `__esri.ViewHit` which can be `__esri.GraphicHit`, `__esri.MediaHit`, or `__esri.RouteHit`. Since we only care about results of type `__esri.GraphicHit`, I just cast the result to that type and then verify it has a `graphic` on it.

Reply:
@cschwinderg type casting is always better than using an `any` type... but instead of type casting, you could do an early return check on the `type` attribute:
```diff
const matches = res.results.filter((item) => {
-    const result = item as __esri.GraphicHit;
-    if(!result.graphic) return null;
+    if (item.type !== 'graphic') return null;
    /* from here on out, the TypeScript compiler narrows `item` from a `ViewHit` down to a `GraphicHit` so you can safely access the `graphic` attribute of `item` directly (so no need for declaring an intermediate type casted variable)
```

Reply:
That was what I tried initially, but it didn't work.

![image](https://user-images.githubusercontent.com/46329268/185212308-0f788ec9-dc4b-4aed-b422-dca01dd556ed.png)

Reply:
You're checking the `graphic` attribute...check the `type` attribute.

Reply:
Good call! I updated it to the simpler method you suggested.